spec/index.bs

```diff
@@ -2517,6 +2524,44 @@ dictionary GPUImageBitmapCopyView {
     </div>
 </div>
 
+### <dfn method for=GPUCommandEncoder>updateBuffer(source, sourceOffset, destination, destinationOffset, size)</dfn> ### {#GPUCommandEncoder-updateBuffer}
+
+<div algorithm="GPUCommandEncoder.updateBuffer">
+
+  **Arguments:**
+    - {{ArrayBuffer}} |source|
+    - {{GPUSize64}} |sourceOffset|
+    - {{GPUBuffer}} |destination|
+    - {{GPUSize64}} |destinationOffset|
+    - {{GPUSize64}} |size|
+
+  **Returns:** void
+
+  Embed a copy of |source| from |sourceOffset| to |size| into the {{GPUCommandEncoder}}.
+  Encode a command into the {{GPUCommandEncoder}} that copies |size| bytes of data from embedded copy to the |destinationOffset| of another {{GPUBuffer}} |destination|.
+  |size| must be less than or equal to 65536 bytes.
```

Comment:
Possibly implementable with Metal's set{,Vertex,Fragment}Bytes which are limited to (or recommended to be under?) 4k.

Reply:
setBytes is only transient. It's basically for setting uniforms OpenGL-style.

Reply:
You're right. I guess those are only useful for a push-constant-like feature.